deno.jsonc

```diff
@@ -1,22 +1,49 @@
 {
-  "lock": false,
-  "exclude": ["dist"],
-  "tasks": {
-    "lint": "deno lint && deno fmt --check",
-    "fmt": "deno fmt",
-    "test": "deno test --doc --allow-run=deno --allow-env --allow-read=./  --allow-write=./ --ignore=./CHANGELOG.md --parallel",
-    "check:examples": "deno check examples/**/*.ts",
-    "snapshot": "deno task test -- --update",
-    "coverage": "deno task test --coverage=./dist/coverage/all/result && deno coverage --lcov ./dist/coverage/all/result > ./dist/coverage/all/cov.lcov",
-    "coverage:ansi": "deno task test ansi --coverage=./dist/coverage/ansi/result && deno coverage --lcov ./dist/coverage/ansi/result > ./dist/coverage/ansi/cov.lcov",
-    "coverage:command": "deno task test command --coverage=./dist/coverage/command/result && deno coverage --lcov ./dist/coverage/command/result > ./dist/coverage/command/cov.lcov",
-    "coverage:flags": "deno task test flags --coverage=./dist/coverage/flags/result && deno coverage --lcov ./dist/coverage/flags/result > ./dist/coverage/flags/cov.lcov",
-    "coverage:keycode": "deno task test keycode --coverage=./dist/coverage/keycode/result && deno coverage --lcov ./dist/coverage/keycode/result > ./dist/coverage/keycode/cov.lcov",
-    "coverage:keypress": "deno task test keypress --coverage=./dist/coverage/keypress/result && deno coverage --lcov ./dist/coverage/keypress/result > ./dist/coverage/keypress/cov.lcov",
-    "coverage:prompt": "deno task test prompt --coverage=./dist/coverage/prompt/result && deno coverage --lcov ./dist/coverage/prompt/result > ./dist/coverage/prompt/cov.lcov",
-    "coverage:table": "deno task test table --coverage=./dist/coverage/table/result && deno coverage --lcov ./dist/coverage/table/result > ./dist/coverage/table/cov.lcov",
-    "coverage:testing": "deno task test testing --coverage=./dist/coverage/testing/result && deno coverage --lcov ./dist/coverage/testing/result > ./dist/coverage/testing/cov.lcov",
-    "update": "deno run --allow-read=./ --allow-net --allow-write=./ https://deno.land/x/deno_outdated@0.2.5/cli.ts --ignore README.md CHANGELOG.md CONTRIBUTING.md"
-    // "update": "deno run --allow-read=./ --allow-write=./ https://deno.land/x/udd@0.8.2/main.ts" globs are a bit weird in tasks: https://github.com/denoland/deno/discussions/15625
+	"name": "@c4spar/cliffy",
```

Comment:
I think i would prefere to use the `@cliffy` scope and publish each module separatly.

- `@cliffy/command`
- `@cliffy/prompt`
- ...

We can create a separat deno.json file in each module with `name`, `version` and `exports` and reference each module in the main `deno.json` file in the `workspaces` property.

```json
{
  "workspaces": [
    "./command",
    "./prompt"
  ]
}
```

We can then run `deno publish` to publish all modules at once.